spec/models/registration_spec.rb

```diff
@@ -22,4 +22,18 @@
       end
     end
   end
+  describe 'destroy_purchased_tickets' do
+    it 'destroys purchased tickets' do
+      user = build(:user)
+      conference = build(:conference)
+      registration1 = create(:registration, conference: conference, user: user)
+      ticket_purchase = build(:ticket_purchase, conference: conference, user: user)
+      ticket_purchase.save
```

Comment:
As discussed in IRC: Please merge these two lines into one and fix the ``useless assigment to variable`` offense reported by rubocop!